app/controllers/application_controller.rb

```diff
@@ -1,5 +1,12 @@
 class ApplicationController < ActionController::Base
   include SetupDeviseBehavior
+  include Pundit::Authorization
 
-  before_action :authenticate_user!
+  rescue_from Pundit::NotAuthorizedError, with: :user_not_authorized
+
+  private
+    def user_not_authorized
+      flash[:alert] = "You are not authorized to perform this action."
```

Comment:
Can you add a translation for this?

Reply:
My Idea is to create a file like this:
`config/locales/policy.xx.yml`

any suggestion?

Reply:
you can put it in globals